guides/v2.3/release-notes/backward-incompatible-changes/index.md

```diff
@@ -11,6 +11,58 @@ View a detailed list of PHP code changes that were made in the "2.3-develop" bra
 - [{{site.data.var.ce}} changes]({{ page.baseurl }}/release-notes/backward-incompatible-changes/open-source.html)
 - [{{site.data.var.ee}} changes]({{ page.baseurl }}/release-notes/backward-incompatible-changes/commerce.html)
 
+## API changes
+
+### StoreManager instead of StoreResolve
+
+`\Magento\Store\Api\StoreResolverInterface` was deprecated.
+
+Instead, use `\Magento\Store\Model\StoreManagerInterface`.
+
+When resolving a store for frontend, use `\Magento\Store\Model\StoreManagerInterface::getStore`.
+
+Do not use `\Magento\Store\Api\StoreResolverInterface::getCurrentStoreId`.
+
+Example of the diff in `StatusBaseSelectProcessor.php` (`Magento\Catalog\Model\ResourceModel\Product`) after update:
+
```

Comment:
The following example shows the diff in `StatusBaseSelectProcessor.php` script after replacing the deprecated method.